lightning/src/onion_message/async_payments.rs

```diff
@@ -131,7 +131,13 @@ pub enum AsyncPaymentsMessage {
 ///
 /// [`Offer::paths`]: crate::offers::offer::Offer::paths
 #[derive(Clone, Debug)]
-pub struct OfferPathsRequest {}
+pub struct OfferPathsRequest {
+	/// The "slot" in the static invoice server's database that this invoice should go into. This
+	/// allows us as the recipient to replace a specific invoice that is stored by the server, which
+	/// is useful for limiting the number of invoices stored by the server while also keeping all the
+	/// invoices persisted with the server fresh.
+	pub invoice_slot: u16,
```

Comment:
"is 65536 enough for anyone"? Given tcp MTU, is there any reason to not just use u64 here?

Reply:
I think I'd normally just not use a relatively small value when it doesn't matter much, as a precaution for unforeseen use cases. Maybe there is a use where a ton of offers with different parameters/amount is pre-generated, and qr codes displayed on website, all while the receiver is offline. Or different offers that encode some metadata that can be useful to track back payments to where a qr was displayed.

But agreed that for our current amountless use case, there is no need to change. Not buying "pre-existing" for something that hasn't been released yet though. EIther way, no blocker.